modules/api/src/main/java/org/apache/ignite/compute/JobDescriptor.java

```diff
@@ -79,19 +92,29 @@ public static Builder builder(String jobClassName) {
      *
      * @return Job descriptor builder.
      */
-    public static Builder builder(Class<? extends ComputeJob<?>> jobClass) {
+    public static Builder builder(Class<? extends ComputeJob<?, ?>> jobClass) {
```

Comment:
Builder class should have the same generic type arguments as the parent class to avoid wildcards.